apps/src/constants.js

```diff
@@ -58,6 +58,7 @@ exports.TestResults = {
   TOO_MANY_BLOCKS_FAIL: 20,               // More than the ideal number of blocks were used.
   APP_SPECIFIC_ACCEPTABLE_FAIL: 21,       // Application-specific acceptable failure.
   MISSING_RECOMMENDED_BLOCK_FINISHED: 22, // The level was solved without a recommended block
+  PASS_WITH_EXTRA_TOP_BLOCKS: 23,         // There was more than one top-level block.
```

Comment:
Hah, yeah, for better or for worse, this feature went through several transformations.